lib/modules/datasource/cdnjs/index.ts

```diff
@@ -98,6 +69,43 @@ export class CdnJsDatasource extends Datasource {
     return val;
   }
 
+  @cache({
+    namespace: `datasource-${CdnJsDatasource.id}-sri`,
```

Comment:
```suggestion
    namespace: `datasource-${CdnJsDatasource.id}-digest`,
```
we should be consistent with other data sources.